apps/admin/backend/schema.sql

```diff
@@ -80,11 +80,19 @@ create table printed_ballots (
     on delete cascade
 );
 
+create table system_settings (
```

Comment:
Note we already have a settings table. I considered adding system settings to settings but didn't to avoid locking us in to making system settings a singleton. There's a case to be made that `system settings` should be renamed something else to avoid confusion, which I would gladly do if there's a clearly better option.